### ISSUE TITLE
Low contrast ratio in marker navigation widget

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/vscode#88126]
* have a file with an error
* press F8
* in our default light theme, the contrast ratio is `2.9:1`

![image](https://user-images.githubusercontent.com/5047891/71809208-e33ab500-306f-11ea-9bd5-cc920894f8db.png)

---

<details><summary>OLD DESCRIPTION from @InduPriya1805:</summary>

**Environment Details:​**
www.typescriptlang.org/​​
​​
**Browser Details: ​​​**
Microsoft Edge 44.18362.1.0, Google Chrome 75.0.3770.142​
​OS: Windows (Version 1903, Build: 18362.113)​​
​
**Repro Steps: ​​​**
1) Hit the URL "www.typescriptlang.org"​​
2) Navigate till "Playground" link and press enter​​
3) Tab till editor field and type any wrong code. ​
4) Press F8 or Shift+F8 to open the errors​
5) Verify the color contrast ratio for text "1 of 2 problems" that is present at the error

**Actual Result: ​​​**
Color contrast ratio for text "1 of 2 problems"  is 2.9:1 which is less than 4.5:1
​​
**Expected Result: ​​​**
Color contrast ratio for text "1 of 2 problems"  should be greater than or equal to  4.5:1
​​
**User Impact: ​​​**
Low vision users will find difficult to read the content if color contrast ratio is less than 4.5:1

**Attachment:**
![Bug_1749_color contrast](https://user-images.githubusercontent.com/42638450/71725125-e27c0600-2e58-11ea-8208-95e364ddeaf0.png)

</details>

**Comment by misolori** [comment]
This now passes:

<img width="50%" src="https://user-images.githubusercontent.com/35271042/72832107-11f08800-3c39-11ea-9d05-f2d0cdb08bba.png" />
